examples/pytorch/bgnn/BGNN.py

```diff
@@ -0,0 +1,356 @@
+import itertools
+import time
+import numpy as np
+import torch
+
+from catboost import Pool, CatBoostClassifier, CatBoostRegressor, sum_models
+from tqdm import tqdm
+from collections import defaultdict as ddict
+import pandas as pd
+from sklearn import preprocessing
+import torch.nn.functional as F
+from sklearn.metrics import r2_score
+
+class BGNNPredictor:
+    def __init__(self,
+                 gnn_model,
+                 task = 'regression',
+                 loss_fn = None,
+                 trees_per_epoch = 10,
+                 backprop_per_epoch = 10,
+                 lr=0.01,
+                 append_gbdt_pred = True,
+                 train_input_features = False,
+                 gbdt_depth=6,
+                 gbdt_lr=0.1):
```

Comment:
Could you pls add docstring for these? We follow the [numpy doc style](https://numpydoc.readthedocs.io/en/latest/format.html). [Code example in DGL](https://github.com/dmlc/dgl/blob/master/python/dgl/nn/pytorch/conv/graphconv.py#L141). It's good to have some code examples for usage.

Reply:
Updated.